doc/api/dns.md

```diff
@@ -140,6 +140,12 @@ changes:
     flags may be passed by bitwise `OR`ing their values.
   - `all` {boolean} When `true`, the callback returns all resolved addresses in
     an array. Otherwise, returns a single address. Defaults to `false`.
+  - `verbatim` {boolean} When `true`, the callback receives IPv4 and IPv6
+    addresses in the order the DNS resolver returned them.  When `false`,
+    IPv4 addresses are placed before IPv6 addresses.
+    Default: currently `false` (addresses are reordered) but this is expected
+    to change in the not too distant future.
+    New code should use `{ verbatim: true }`.
```

Comment:
I don't have a strong opinion on preferring `verbatim: true`, but shouldn't we add a runtime deprecation of the old default?